src/app/containers/PageViewAnalytics/labelHelpers/index.test.js

```diff
@@ -214,11 +214,20 @@ describe('getCurrentTime', () => {
 
   it('should return hours, mins and seconds joined by "x"', () => {
     const date = new Date(946729425000); // 12:23:45
+    const currentTimeZoneOffsetInHours = date.getTimezoneOffset() / 60;
+    date.setHours(date.getHours() - currentTimeZoneOffsetInHours);
```

Comment:
Why do you need to do this? these functions should return the exact same in the code. I would rather you mock `getHours`, `getMinutes` and `getSeconds` to just return a number

Reply:
I see your point. But if you were to mock `getHours`, `getMinutes` and `getSeconds` then there is no point in the original test setting `date` on the window.